### DISCUSSION TITLE
Add config setting to make add to queue reply ephemeral

### DISCUSSION THREAD

**Discussion by Sheeley7** [codetheweb/muse#1019] [Ideas]
I'd like to avoid spamming the chat with /play request replies from the bot. It would be super helpful if the add-query-to-queue.ts interaction.deferReply(); had a config setting to make it ephemeral like interaction.deferReply({ ephemeral: config.EPHEMERAL_REPLY });

**Comment by codetheweb** [comment]
happy to review a PR for this 👍

**Reply by Sheeley7** [reply]
Could you point me in the direction of how your configs are setup?

**Comment by codetheweb** [comment]
https://github.com/codetheweb/muse/pull/1021
